runtime/firecracker-runc-config.json.example

```diff
@@ -0,0 +1,155 @@
+{
+    "ociVersion": "1.0.1-dev",
+    "process": {
+        "terminal": false,
+        "user": {
+            "uid": 0,
+            "gid": 0
+        },
+        "env": [
+            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+            "TERM=xterm"
+        ],
+        "cwd": "/",
+        "capabilities": {
+            "bounding": [
+            ],
+            "effective": [
+            ],
+            "inheritable": [
+		"CAP_NET_ADMIN",
+		"CAP_MKNOD"
+            ],
+            "permitted": [
+		"CAP_NET_ADMIN",
+		"CAP_MKNOD"
+            ],
+            "ambient": [
+		"CAP_NET_ADMIN",
+		"CAP_MKNOD"
+            ]
+        },
+        "rlimits": [
+            {
+                "type": "RLIMIT_NOFILE",
+                "hard": 1024,
+                "soft": 1024
+            }
+        ],
+        "noNewPrivileges": true
+    },
+    "root": {
+        "path": "rootfs",
+        "readonly": false
+    },
+    "hostname": "runc",
+    "mounts": [
+        {
+            "destination": "/proc",
+            "type": "proc",
+            "source": "proc"
+        },
+        {
+            "destination": "/dev/pts",
```

Comment:
Does firecracker need a pseudoterminal at any point? I'd try removing this if you haven't already to see what happens. It might be something runc itself actually wants there, but if not and firecracker works without it we may as well remove this.

Same comment for `/dev/shm` and `/dev/mqueue` below, I'm not sure if firecracker/runc actually have a dependency on it.